ipam/allocator.go

```diff
@@ -413,6 +418,10 @@ func (a *Allocator) getPredefinedPool(as string, ipV6 bool) (*net.IPNet, error)
 		}
 	}
 
+	if !supported {
```

Comment:
hmm. wont this check cause a legitimate pool exhaustion mis-categorized as `ErrNotSupported` ?